librz/core/cmd_type.c

```diff
@@ -1883,6 +1919,84 @@ RZ_IPI RzCmdStatus rz_type_define_handler(RzCore *core, int argc, const char **a
 	return RZ_CMD_STATUS_OK;
 }
 
+RZ_IPI RzCmdStatus rz_type_list_enum_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+	const char *enum_name = argc > 1 ? argv[1] : NULL;
+	// TODO: Reconsider the `te <enum_name> <member_value>` syntax change
+	const char *member_value = argc > 2 ? argv[2] : NULL;
+	if (enum_name) {
+		if (member_value) {
+			return types_enum_member_find(core, enum_name, member_value);
+		} else {
+			types_enum_print(core, enum_name, mode);
+		}
+	} else {
+		// A special case, since by default `te` returns only the list of all enums
```

Comment:
can't we just add support for `te` and `teq`?

Reply:
I am not sure about this to be honest. It makes sense to list only enum names by default, since there can be many.

Reply:
Right, but maybe we can consider STANDARD just the enum names and LONG the C version. This way you'd have just one command to list enums in different formats:
`te` -> just enum names (STANDARD)
`tej` -> enum in json (JSON)
`tek` -> enum in sdb (SDB)
`tel` -> enum in C (LONG)

Reply:
But then `te <enum name>` command wouldn't have any meaning like right now. Other types have output with `pf` string when you call e.g. `tu <union name>`.

Reply:
So many inconsistencies :( I even wonder why one would want to print "pf format" instead of something easier... I like more and more the idea of having `:<mode>` as a way to specify the output mode you want. So commands could just have `<cmd>:pf` or `<cmd>:sdb` `<cmd>:json` instead of having to find a letter for each case, ending up with many inconsistencies :(

Reply:
Yes, the idea of having the ":<mode>" is the good one.